packages/scandipwa/src/component/Breadcrumbs/Breadcrumbs.component.js

```diff
@@ -63,7 +65,8 @@ export class Breadcrumbs extends PureComponent {
 
         if (
             !areBreadcrumbsVisible
-            || pathname.match(appendWithStoreCode('/account'))
+            || pathname.match(appendWithStoreCode(ACCOUNT_URL))
+            || pathname.match(appendWithStoreCode(CHECKOUT_URL))
             || isHomePageUrl(pathname)
```

Comment:
This condition has become too complex.
Please move it to a separate variable or method.

Reply:
Done in the next commit